test/percy/local-theme-provider.jsx

```diff
@@ -0,0 +1,42 @@
+import { useRef } from 'react';
+import PropTypes from 'prop-types';
+import { ThemeProvider } from '@commercetools-uikit/design-system';
+
+export const LocalThemeProvider = (props) => {
+  const ref = useRef(null);
+
+  return (
+    <div ref={ref}>
+      <ThemeProvider
+        ref={ref}
+        theme={props.theme}
+        customPropertiesOverrides={props.customPropertiesOverrides}
+      >
+        {props.children}
+      </ThemeProvider>
+    </div>
+  );
+};
+LocalThemeProvider.propTypes = {
+  children: PropTypes.node.isRequired,
+  customPropertiesOverrides: PropTypes.object,
+  theme: PropTypes.string,
+};
+
+const darkTheme = {
+  colorSurface: 'black',
+  colorSolid: 'white',
+  colorNeutral60: 'rgba(255,255,255,0.60)',
+  colorNeutral: 'rgba(255,255,255,0.60)',
+  colorAccent98: 'rgba(0,0,0,0.98)',
+};
+
+export const LocalDarkThemeProvider = (props) => (
```

Comment:
I'm not sure why we still need this dark theme theme provider. I thought we agreed on not having themed tests but in the `ThemeProvider`. But maybe I'm wrong.

Reply:
Perhaps I got it wrong. I had in my notes to remove only the custom theme. Removed it 413e7c109f783d0dc0d5b1f4d75dbdd2bf7a0d3a